packages/next/src/server/async-storage/request-async-storage-wrapper.ts

```diff
@@ -45,10 +51,26 @@ function getMutableCookies(
   return MutableRequestCookiesAdapter.wrap(cookies, onUpdateCookies)
 }
 
+export type WrapperRenderOpts = RenderOpts &
+  Partial<RequestLifecycleOpts> &
+  RenderOptsForRouteHandlerPartial &
+  RenderOptsForWebServerPartial
+
+type RenderOptsForRouteHandlerPartial = Partial<
+  Pick<
+    RenderOpts,
+    'ComponentMod' // undefined in a route handler
+  >
+>
+
+type RenderOptsForWebServerPartial = {
+  experimental: Pick<RenderOpts['experimental'], 'after'>
```

Comment:
I prefer if we can only pass down an explict argument `after` or `isNextAfterEnabled` as a boolean value everywhere instead of creating a new type of `experimental`, which might lead to confusion and absue of this flag.